### PR TITLE
Adding a few more cores for new systems

### PR DESCRIPTION
**Pull request by telanus**
Added the PC-98 and the X68000 to the RA runner

### CODE REVIEW

**Comment by RobLoach** [comment]
`citra` and `dolphin`, while you're at it :wink:

File: lutris/runners/libretro.py [hunk 1]
```diff
@@ -94,6 +99,7 @@ class libretro(Runner):
         'Sharp X68000',
         'Sony PlayStation Portable',
         'Sega Dreamcast',
+        'Sega Dreamcast',
```
Comment (TingPing) [inline lutris/runners/libretro.py]: Duplicate?

File: lutris/runners/libretro.py [hunk 2]
```diff
@@ -11,8 +11,10 @@ def get_core_choices():
     # The order has to be the same!
     return [
         ('4do (3DO)', '4do'),
+        ('Citra (Nintendo 3DS)', 'citra'),
         ('CrocoDS (Amstrad CPC)', 'crocods'),
         ('DesmuME (Nintendo DS)', 'desmume'),
+        ('Dolphin (Nintendo Wii)', 'dolphin'),
```
Comment (TingPing) [inline lutris/runners/libretro.py]: and GameCube?

Comment (RobLoach) [inline lutris/runners/libretro.py]: `Dolphin (Nintendo Wii/Gamecube)`?

File: lutris/runners/libretro.py (hunk 1 above)
Comment (telanus) [inline lutris/runners/libretro.py]: nope, redream is a dreamcast emu and reicast is a dreamcast emu, and red comes before rei, tried to keep to the alphabetic order that was already there in the core election order, thus here it look like a double entry when it isn't really

**Pull request merged**